alien4cloud/topology.go

```diff
@@ -517,3 +518,55 @@ func (t *topologyService) SaveA4CTopology(ctx context.Context, a4cCtx *TopologyE
 
 	return processA4CResponse(response, nil, http.StatusOK)
 }
+
+func (t *topologyService) GetAllTopologies(ctx context.Context) ([]BasicTopologyInfo, error) {
```

Comment:
To have a more generic API similar to the REST API, we could call the function `GetTopologies()` and add the argument `query string` that you would pass below in the searchRequest.query parameter.

Reply:
Sounds good, done.